spec/models/sale_price_spec.rb

```diff
@@ -39,7 +39,7 @@
     end
 
     it 'sets the end time' do
-      expect(sale_price.end_at).to be_within(1.second).of(Time.now)
+      expect(sale_price.end_at).to be_within(2.second).of(Time.now)
```

Comment:
I don't actually care if you fix, but fwiw in the commit message "accomidate" should be "accommodate".

Reply:
Thank you for catching that, I will get it rectified.